communication/slack-config/channels.yaml

```diff
@@ -84,7 +84,8 @@ channels:
   - name: gitops
   - name: gke
   - name: gloo
-  - name: gsoc-apps
```

Comment:
@nikhita where the GSoC communication will happen in this case?

I understand the rationale to save the conversations' history, but it's not clear why a new channel is needed (instead of creating a new one in parallel).

Reply:
> @nikhita where the GSoC communication will happen in this case?

I guess we decided to rename because there won't be a gsoc session after the summer. But yeah, I think it would still be useful to have a "gsoc" apps channel too. Can we rename this to `#gsoc-outreachy-apps`? Thoughts?

I'd prefer not to have a whole new channel for outreachy because the conversations are similar and asking folks to keep track of multiple channels is very cumbersome, especially when we already have problems with low volunteers. :(

Reply:
@nikhita we may another GSoC round next year, not sure if we'll want to rename the channel back :)

Also, we may have other mentoring programs besides these two (eg. Community Bridge). How about creating a #mentoring or #mentoring-programs channel that will cover all of them (leaving #gsoc as-is).

I'm not a big fan of renaming - the contexts are similar, but different. Instead, I'd propose to use the new channel for the mentoring programs, slightly moving the conversations out of the current channel.